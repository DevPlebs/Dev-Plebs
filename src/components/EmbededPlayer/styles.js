import tw from "tailwind.macro"

export const StyledEmbededPlayer = tw.iframe`border-none`
