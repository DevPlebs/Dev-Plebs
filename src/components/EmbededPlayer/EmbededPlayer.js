import React from "react"
import { string } from "prop-types"
import { StyledEmbededPlayer } from "./styles"

export const EmbededPlayer = ({ title, mediaUrl }) => (
  <StyledEmbededPlayer
    title={title}
    src={`${mediaUrl}&?from=usersite&vjs=1&skin=1&fonts=Helvetica&auto=0&download=1`}
    height="400"
    width="400"
    scrolling="no"
    data-name="pb-iframe-player"
  ></StyledEmbededPlayer>
)

EmbededPlayer.propTypes = {
  title: string,
  mediaUrl: string,
}

export default EmbededPlayer
