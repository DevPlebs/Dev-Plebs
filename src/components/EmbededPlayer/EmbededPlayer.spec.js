import React from "react"
import { create } from "react-test-renderer"

import EmbededPlayer from "./EmbededPlayer"

describe("EmbededPlayer", () => {
  it("renders correctly", () => {
    const tree = create(<EmbededPlayer />)
    expect(tree.toJSON()).toMatchSnapshot()
  })
})
