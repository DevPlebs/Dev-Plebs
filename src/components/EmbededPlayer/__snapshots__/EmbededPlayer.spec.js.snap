// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmbededPlayer renders correctly 1`] = `
.emotion-0 {
  border-style: none;
}

<iframe
  className="emotion-0"
  data-name="pb-iframe-player"
  height="400"
  scrolling="no"
  src="undefined&?from=usersite&vjs=1&skin=1&fonts=Helvetica&auto=0&download=1"
  width="400"
/>
`;
